talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -53,6 +53,10 @@
     "   * Manuscript: [_Nucleic Acids Res._ (2010), __38__, W724-31](https://academic.oup.com/nar/article/38/suppl_2/W724/1122840)\n",
```

Comment:
<p>Well done. This is again a nice summary covering this complex topic on a level suitable for TeachOpenCADD talktorials.</p><p> </p><p>Remarks:</p><p> </p><p>Sampling methods: if 4. and 5. are examples for 3. - maybe they should be 3.1 and 3.2 instead?</p><p> </p><p>Point 3 on scoring functions is still in German :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>